File: Backend/src/main/java/com/tarikturkdil/dto/NotificationResponse.java
package com.tarikturkdil.dto;

import com.tarikturkdil.entity.NotificationType;

import lombok.Data;

@Data
public class NotificationResponse extends DtoBase{

	private String actorName;
	private String actorAvatarUrl;
	private NotificationType type;
	private Long referenceId;
	private boolean isRead;
}

File: Frontend/app/src/main/java/com/tarikturkdil/photoProject/data/remote/dto/NotificationResponse.kt
package com.tarikturkdil.photoProject.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class NotificationResponse(
    val id: Long,
    val actorName: String? = null,
    val actorAvatarUrl: String? = null,
    val type: NotificationType,
    val referenceId: Long? = null,
    val read: Boolean = false,
    val createTime: String? = null
)